motionplan/tpspace/ptgGroupFrame.go

```diff
@@ -102,6 +104,12 @@ func NewPTGFrameFromKinematicOptions(
 			// If max allowed angular velocity would turn tighter than given turn radius, shrink the max used angular velocity
 			// to match the requested tightest turn radius.
 			angVelocityRadps = velocityMMps / turnRadMillimeters
+			// This is a warning message because the user will observe the base turning at a different speed than the one requested.
+			logger.Warnf(
+				"given turning radius was %f but a linear velocity of %f "+
+					"meters per sec and angular velocity of %f degs per sec would turn at a radius of %f. Decreasing angular velocity to %f.",
+				turnRadMeters, velocityMMps/1000., angVelocityDegps, calcTurnRadius, rdkutils.RadToDeg(angVelocityRadps),
```

Comment:
why are we dividing `velocityMMps` by 1000? The naming convention implies to me that `velocityMMps` is in millimeters & is being compared to a value in meters (turnRadMeters).

Reply:
The reason is that while motion planning uses mm internally, the unit input by the user in the nav service is in meters per second, so we display the warning message in terms of the exact values the user entered, rather than a transformation of them.